test/e2e/e2e_test.go

```diff
@@ -242,8 +245,12 @@ var _ = ginkgo.Describe("Kueue visibility server", func() {
 					info, err := visibilityClient.ClusterQueues().GetPendingWorkloadsSummary(ctx, clusterQueue.Name, metav1.GetOptions{})
 					gomega.Expect(err).NotTo(gomega.HaveOccurred())
 					return info.Items
-					// We do not check Name as it's generated for workloads
-				}, util.Timeout, util.Interval).Should(gomega.BeComparableTo(wantPendingWorkloads, cmpopts.IgnoreFields(metav1.ObjectMeta{}, "Name")))
+					// We do not check Name and OwnerReference's UID as they are generated
+				}, util.Timeout, util.Interval).Should(gomega.BeComparableTo(
+					wantPendingWorkloads,
+					cmpopts.IgnoreFields(metav1.ObjectMeta{}, "Name"),
```

Comment:
Can we create the const / var for the `cmpopts` as pendingWorkloadsCmpOpts, then move the comment to the definition of the const, rather than duplicate.